utils/build/docker/nodejs/express4-typescript/iast.ts

```diff
@@ -348,6 +350,34 @@ function initSinkRoutes (app: Express): void {
     eval('1+2')
     res.send('OK')
   })
+
+  app.post('/iast/template_injection/test_insecure', (req, res) => {
+    const { library, template } = req.body
+    let fn
+
+    if (library === 'pug') {
+      fn = pug.compile(template)
+    } else {
+      fn = handlebars.compile(template)
+    }
+
+    const html = fn()
+    res.send(`OK:${html}`)
+  })
+
+  app.post('/iast/template_injection/test_secure', (req, res) => {
+    const { library } = req.query
```

Comment:
`req.query.library` will always be `undefined`, since we are not passing any query param when doing a request to `test_secure`

Reply:
Good catch

Reply:
Done